common/scripts/package.json

```diff
@@ -5,20 +5,20 @@
   "license": "MIT",
   "repository": "https://github.com/iTwin/admin-components-react/tree/main/common/scripts",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.20.0",
-    "@typescript-eslint/parser": "^4.20.0",
+    "@typescript-eslint/eslint-plugin": "^6.12.0",
```

Comment:
I tried creating a branch to fix an issue but I'm unable to commit to it, there seems to be an issue with the different versions of eslint used here and within Jest which uses 5.62:
<img width="849" alt="image" src="https://github.com/iTwin/admin-components-react/assets/1904889/cdc31bd9-6e78-4e28-89e5-2af292295498">

Reply:
Pull the latest changes and hopefully it'll work otherwise delete the temp folder under common and run the command again

Reply:
Seems ok!